### PR TITLE
Feature/developer docs

### PR DESCRIPTION
**Pull request by glow-mdsol**
@ykitamura-mdsol @jcarres-mdsol @ssteeg-mdsol 
Added components for automatic generation of documentation (@ChazzMaTazz FYI)
This should automatically get deployed to https://requests-mauth.readthedocs.io/en/latest/ once the build is updated.

@isparks FYI

### CODE REVIEW

**Comment by iansparks** [comment]
Looks good. Having my name as author is getting weird though. Maybe a CONTRIBUTORS file and Medidata as author?

**Comment by glow-mdsol** [comment]
Ok, added a CONTRIBUTORS.rst file and updated the documents

**Comment by iansparks** [comment]
Thanks. No further comments. Don't have rights to merge so leaving that to review of @ykitamura-mdsol

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'MauthAuthenticatorforRequests.tex', 'Mauth Authenticator for Requests Documentation',
+     'Ian Sparks', 'manual'),
```
Comment (ykitamura-mdsol) [inline docs/conf.py]: Should we use `author` here also?

**Comment by glow-mdsol** [comment]
Updated - thanks!

**Pull request merged**